docs/windows.md

```diff
@@ -201,6 +201,15 @@ Download and apply `antrea-windows.yml`.
 kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-windows.yml
 ```
 
+Since Antrea 1.10, in kube-proxy mode, you can also deploy antrea-agent Windows DaemonSet with containerd runtime by applying file `antrea-windows-containerd.yml`.
```

Comment:
Why antrea-agent with containerd runtime has dependency on kube-proxy mode?

Reply:
We have ovs core issue with containerd in proxyall mode.